### PR TITLE
Change application ID for RxJava template

### PR DESCRIPTION
**Pull request by Tuubz**
https://github.com/nimblehq/android-templates/projects/3#card-57266839

## What happened 🤔

But please review #63 first 🙏

Since we have a specific package name & application ID for the CouritineTemplate, we should change it accordingly for the RxJavaTemplate as well

## PoW (a.k.a Proof Of Work)💪

App is still buildable 🙏

### CODE REVIEW

**Review by Wadeewee — APPROVED** [review]

File: RxJavaTemplate/app/build.gradle [hunk 1]
```diff
@@ -52,11 +52,11 @@ android {
     flavorDimensions "stage"
     productFlavors {
         production {
-            applicationId "co.nimblehq"
+            applicationIdSuffix ".production"
```
Comment (hoangnguyen92dn) [inline RxJavaTemplate/app/build.gradle:55]: Hmm. Seems the suffix `.production` for production bundle is unnecessary 🤔

Comment (Tuubz) [reply RxJavaTemplate/app/build.gradle:55]: Yeah, sounds good to me 😁

What do you think @Wadeewee ?

Comment (Wadeewee) [reply RxJavaTemplate/app/build.gradle:55]: Yeah, I agree so we just put it like this 👇  I think it enoung for now 🙏 
`flavorDimensions "stage"
    productFlavors {
        production
        staging
    }
`

**Comment by Tuubz** [comment]
Updated 🚀 

But please review #63 first 🙏

**Review by luongvo — APPROVED** [review]
all lgtm 💯

**Review by hoangnguyen92dn — APPROVED** [review]
LGTM 👍

**Comment by hoangnguyen92dn** [comment]
@Tuubz Could you please add the Card's link to the description? 🙏

**Review by lydiasama — APPROVED** [review]

**Comment by Tuubz** [comment]
> @Tuubz Could you please add the Card's link to the description? 🙏

Done 🚀

**Comment by hoangnguyen92dn** [comment]
> > @Tuubz Could you please add the Card's link to the description? 🙏
> 
> Done, I created a new card for this 🚀

@Tuubz Still don't see the card's link in the PR's description 🤔 
image

**Comment by Tuubz** [comment]
My bad, forgot to update the link in this PR as well 🙏 

Done ✅

**Pull request merged**